### DISCUSSION TITLE
Document read-write loadable atoms

### DISCUSSION THREAD

**Discussion by danielr18** [pmndrs/jotai#1436] [Ideas]
I found how to create read-write loadable atoms in this comment https://github.com/pmndrs/jotai/discussions/1061#discussioncomment-2406494
```js
const asyncAtom = atom(...)
const loadableAtom = loadable(asyncAtom)
const finalAtom = atom(
  (get) => get(loadableAtom),
  (get, set, arg) => set(asyncAtom, arg)
)
```

It may save time to others if we add it to the documentation, as you may not know this composition is possible if you are just starting to learn how jotai works.

**Comment by dai-shi** [comment]
we could add it in loadable util api doc, or in advanced recipes. which would you suggest?
(we should split advance recipes into files, anyway)
